packages/styles/src/components/tabs/_tabs-wrapper.scss

```
@@ -0,0 +1,69 @@
+@forward './../../variables/options';
+
+@use './../../mixins/utilities' as utilities-mx;
+@use './../../variables/color';
+@use './../../variables/spacing';
+@use './../../variables/components/nav';
+
+.tabs-wrapper {
+  position: relative;
+  padding-top: spacing.$spacer;
+  border: 0;
+  background-color: color.$gray-background-light;
+
+  // Create a line that lies below the active but above the passive elements
+  // This way hover works smoothly with the background color
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 1px;
+    background-color: nav.$nav-tabs-border-color;
+  }
+
+  // Small hack to save a lot of code and provide a lot of flexibility to tabs coloring
+  .tabs {
+    background-color: transparent !important;
+  }
+
+  @include utilities-mx.high-contrast-mode() {
+    &::after {
+      background-color: ButtonBorder;
+    }
+  }
+}
+
+.tabs {
+  @include utilities-mx.reset-list;
+  display: flex;
+  flex-wrap: nowrap;
+  overflow-x: auto;
+
+  /* prevent scroll chaining on x scroll */
+  overscroll-behavior-x: contain;
+  white-space: nowrap;
+
+  // Scrolling fix, make the tabs scroll a little more so the right most tab is not
```

I think it's missing a word after 'a little more'

---

Sorry, I've misread it I think. Maybe this help to read it:

```suggestion
  // Scrolling fix to make the tabs scroll a little more so the right most tab is not
```
or

```suggestion
  // Scrolling fix: make the tabs scroll a little more so the right most tab is not
```

or

```suggestion
  // Make the tabs scroll a little more so the right most tab is not
```